components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksFeature.kt

```diff
@@ -135,17 +132,19 @@ class AppLinksFeature(
         }
     }
 
-    private fun handleFallback(redirect: AppLinkRedirect, session: Session) {
-        redirect.webUrl?.let {
-            sessionManager.getOrCreateEngineSession(session).loadUrl(it)
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal fun handleFallback(redirect: AppLinkRedirect, session: Session) {
+        redirect.fallbackUrl?.let {
+            val c = sessionManager.getOrCreateEngineSession(session)
+            c.loadUrl(it)
```

Comment:
We should probably make this use `SessionUseCases` instead of like this.

Reply:
issue created to track this since this will probably break the API. https://github.com/mozilla-mobile/android-components/issues/4988